docs/sources/reference/export_import.md

```diff
@@ -29,14 +29,14 @@ the dashboard, and will also be added as an required input when the dashboard is
 
 ## Importing a dashboard
 
-To import a dashboard open dashboard search and then hit the import button.
+To import a dashboard click the + icon in the sidemenu, and click "import". 
```

Comment:
```suggestion
To import a dashboard click the + icon in the side menu, and then click **Import**. 
```

Reply:
Please acquaint yourself with the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md)